source/simulator/core.cc

```diff
@@ -197,7 +197,11 @@ namespace aspect
     dof_handler (triangulation),
 
     rebuild_stokes_matrix (true),
-    rebuild_stokes_preconditioner (true)
+    rebuild_stokes_preconditioner (true),
+
+    // Set rotating checkpointing information
+    checkpoint_index(0),
+    checkpoint_log_file ("checkpoint.log")
```

Comment:
The checkpoint file should probably reside in the output file, where we also keep the checkpoints themselves.